### DISCUSSION TITLE
news/20240225/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#343] [Website Discussions]
# news/20240225/

X2Pro5 v5.0.34 is out now with the following fix:

https://fcp.cafe/news/20240225/

**Comment by dmetz01** [comment]
Anybody else confused by the new checkbox in PBF prefs?

If I've chosen to Add thumbnails, why would I need a second checkbox to keep them from being deleted?

**Reply by dmetz01** [reply]
UPDATE: Heard back from support on this.  The app can now leave copies of the thumbnails in a folder in Finder, in case you want them for some other purpose.
